Quote this to prevent word splitting.

[out script](https://github.com/homedepot/newrelic-deploy-resource/blob/master/out)
    - Line 68

[SC2046](https://github.com/koalaman/shellcheck/wiki/SC2046): Quote this to prevent word splitting.
When command expansions are unquoted, word splitting and globbing will occur. This often manifests itself by breaking when filenames contain spaces

Note: All the other args are quoted but not timestamp	

Source: [ShellCheck](https://www.shellcheck.net/)

--- a/out
+++ b/out
@@ -62,7 +62,7 @@ fi
 
 
 jq -n \
-    --arg ts $(date +%s) \
+    --arg ts "$(date +%s)" \
     --arg revision "${revision}" \
     --arg changelog "${changelog}" \
     --arg description "${description}" \

